content/docs/1.10.1/important-notes/_index.md

```diff
@@ -42,7 +42,63 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
 
 ### `longhorn.io/v1beta1` API
 
-The `v1beta1` Longhorn API version was removed in v1.10.0.
+The `v1beta1` Longhorn API version was removed in v1.10.0. The API migration for old `v1beta1` should be done automatically. However, in certain scenarios, such as when backup system interventions occur during the Longhorn upgrade process, some Custom Resources (CRs) may still remain in the Kubernetes database using the legacy `v1beta1` version. If CR objects are not upgraded to the latest `v1beta2` version, this will prevent the Longhorn manager from functioning properly:
+
+```
+Upgrade failed: cannot patch "backingimagedatasources.longhorn.io" with kind CustomResourceDefinition: CustomResourceDefinition.apiextensions.k8s.io "backingimagedatasources.longhorn.io" is invalid: status.storedVersions[0]: Invalid value: "v1beta1": missing from spec.versions; v1beta1 was previously a storage version, and must remain in spec.versions until a storage migration ensures no data remains persisted in v1beta1 and removes v1beta1 from status.storedVersions
+```
+
+The recommended approach is to actively verify that all `*.longhorn.io` CRDs have been upgraded to the correct version after the system upgrade is complete. All CR objects should be stored as `v1beta2`:
+
+```bash
+kubectl get crd -l app.kubernetes.io/name=longhorn -o=jsonpath='{range .items[*]}{.metadata.name}{": "}{.status.storedVersions}{"\n"}{end}'
+
+backingimagedatasources.longhorn.io: ["v1beta2"]
+backingimagemanagers.longhorn.io: ["v1beta2"]
+backingimages.longhorn.io: ["v1beta2"]
+backupbackingimages.longhorn.io: ["v1beta2"]
+backups.longhorn.io: ["v1beta2"]
+backuptargets.longhorn.io: ["v1beta2"]
+backupvolumes.longhorn.io: ["v1beta2"]
+engineimages.longhorn.io: ["v1beta2"]
+engines.longhorn.io: ["v1beta2"]
+instancemanagers.longhorn.io: ["v1beta2"]
+nodes.longhorn.io: ["v1beta2"]
+orphans.longhorn.io: ["v1beta2"]
+recurringjobs.longhorn.io: ["v1beta2"]
+replicas.longhorn.io: ["v1beta2"]
+settings.longhorn.io: ["v1beta2"]
+sharemanagers.longhorn.io: ["v1beta2"]
+snapshots.longhorn.io: ["v1beta2"]
+supportbundles.longhorn.io: ["v1beta2"]
+systembackups.longhorn.io: ["v1beta2"]
+systemrestores.longhorn.io: ["v1beta2"]
+volumeattachments.longhorn.io: ["v1beta2"]
+volumes.longhorn.io: ["v1beta2"]
+```
+
+If any CR objects still exist in the system using the legacy `v1beta1` version, manual version migration must be triggered. The following script can help upgrade all objects in the system:
+
+```bash
+kubectl get validatingwebhookconfigurations longhorn-webhook-validator -o yaml >longhorn-validating-webhook.yaml
+kubectl get mutatingwebhookconfigurations longhorn-webhook-mutator -o yaml >longhorn-mutating-webhook.yaml
+kubectl delete validatingwebhookconfigurations longhorn-webhook-validator
+kubectl delete mutatingwebhookconfigurations longhorn-webhook-mutator
```

Comment:
Temporarily disabling webhooks is still needed. Some CRs like `stable-longhorn-versions` setting is read-only, and the validator will reject to attach the migration annotation.